### DISCUSSION TITLE
Nirvana NV14 with analog gimbals/potis

### DISCUSSION THREAD

**Discussion by tkilla77ffm** [EdgeTX/edgetx#1157] [Q&A]
Hello,
I would like to use analog gimbals or potis in my Nirvana NV14 instead of the hall gimbals.
It looks like there are connectors for CH1/CH2 and CH3/CH4 on board. I connected analog gimbals on those ports, with the serial port from the hall gimbals disconnected, but it didnt detect the analog gimbals. For some radios, after 70ms without detecting a hall gimbal,
it switches to analog reading.

can somebody help me out?

thnx a lot

**Comment by pfeerick** [comment]
You can try reverting this commit (https://github.com/EdgeTX/edgetx/pull/1054/commits/d8237a9f50b8cdec20c331bd605c67183cca5149) / setting `globalData.flyskygimbals = false;`, as it is specifically overridden on the NV14, preventing the auto-detection from switching.

**Reply by tkilla77ffm** [reply]
thank you very much. This would solve my problem. 
The next problem is, im not  yet in with this coding stuff. Is it possible to if you can make me a ready to use FW? ... with the "false" statement?

**Reply by pfeerick** [reply]
Sure... as long as you don't mind it being a main branch (i.e. nightly): [NV14_907f730_flyskygimbals=false.zip](https://github.com/EdgeTX/edgetx/files/7605637/NV14_907f730_flyskygimbals.false.zip)

**Reply by tkilla77ffm** [reply]
thanks a lot peter,
i hope to have sucsess with my project, then i would be glad to show you

thank you very much

**Reply by tkilla77ffm** [reply]
you mean it is not specially for the NV14?

**Reply by pfeerick** [reply]
It is specially for the NV14, but based on the main branch - i.e. the development branch leading to v2.6.

**Reply by tkilla77ffm** [reply]
i understand, thank you

**Reply by tkilla77ffm** [reply]
hello peter,
i`m sorry to bother you again. The version you gave me dosnt work wright for a reason.
First of all, in this version, the touch screen is very sensitiv, a single touch causes somtimes multiple inputs.
Also, in this version, the analog inputs dosnt work. In the channel monitor, CH1 - CH4 is twitching around the center.
Calibrating doesnt work either, because the analog gimbals/input are not recognized.
Connecting the hall gimbals again (then boot up), the same problem occurs. Twitching around the center, no calibration possible.

with the latest nightly, the hall gimbals works fine after calibrating and the analog inputs are disabled.
But the touch screen is very sensitiv as well.

With the v2.5.0, everything works fine, exept the analog inputs which i want to use for a reason.

do you have any clou or idea?

**Reply by pfeerick** [reply]
The hall gimbals not working at all is to be expected, because changing that line to false basically disables them completely as autodetection code didn't work on NV14 anyway. Touch behaving differently is to be expected - is development branch build after all ;) Analog sticks not working, I have no idea atm - it may not be a simple change as NV14 was always hall effect sticks AFAIK, so everything specific to it will be coded for it with that in mind.

**Reply by tkilla77ffm** [reply]
i got it,
thanks again
